README.md

```diff
@@ -205,18 +205,19 @@ Except for resume uploading, all of the fields are available in both JSON mode
 and multipart form-data mode. The name and email address fields are both
 required. The candidate will be emailed after they apply to the job, unless the `silent` field is set to true.
 
-
-| Field             | Description                   |
-| ----------------- | ----------------------------- |
+| Field               | Description                   |
+| ------------------- | ----------------------------- |
 | `name` (*required*) | Candidate's name
-| `email` (*required*)| Email address. Requires an @-symbol. Candidate records will be merged when email addresses match. 
+| `email` (*required*)| Email address. Requires an @-symbol. Candidate records will be merged when email addresses match.
 | `resume`            | Resume data. Only in `multipart/form-data` mode. Should be a file.
 | `phone`             | Phone number
 | `org`               | Current company / organization
 | `urls`              | URLs for sites (Github, Twitter, LinkedIn, Dribbble, etc). Should be a JSON map or individual urls[GitHub], urls[Twitter], etc fields
 | `comments`          | Additional information from the candidate
 | `silent`            | Disables confirmation email sent to candidates upon application. Accepts values of true, false, "true" or "false".
-| `source`             | Adds a source tag to candidate (e.g. 'LinkedIn')
+| `source`            | Adds a source tag to candidate (e.g. 'LinkedIn')
+| `ip` (*in beta*)    | Comma-separated string of IP addresses that the application is routed through, starting with the applicant's IP address. (e.g. `"ip":"12.345.678.901,23.456.789.012"`)
+| `consent` (*in beta*) | Object of consent statuses. Currently, `marketing` is the only consent status and is expected to be a boolean. (e.g. `"consent":{"marketing":true}`)
```

Comment:
The main changes are lines `219` and `220` - not sure if we should add a note above the table that calls out `ip` and `consent` to give a bit more context about when/why you'd use these two fields.

Also unsure if the `(*in beta*)` note should live here or elsewhere.